staging/src/k8s.io/component-base/version/verflag/verflag.go

```diff
@@ -34,9 +36,16 @@ const (
 	VersionFalse versionValue = 0
 	VersionTrue  versionValue = 1
 	VersionRaw   versionValue = 2
+	VersionJson  versionValue = 3
```

Comment:
Looks a little odd to me for the constants to be public but the type is still private. Is that common?

Reply:
It's also uncommon in Kubernetes APIs to use numeric enums; instead they usually just rely on string constants. But it looks like we were already using int enum here...

Reply:
> Looks a little odd to me for the constants to be public but the type is still private. Is that common?

True, I just appended new values, trying to avoid "architectural changes". I can change the values to be private if that is desired in this PR. I can also create a separate PR about this.

> But it looks like we were already using int enum here

Yeah, I didn't even think about this. The numeric constants could be dropped if using just string values. I'd prolly do this in a separate PR, though(?)

Reply:
I try to address the public/private issue in #88231